internal/pkg/version/check_manifest_test.go

```diff
@@ -0,0 +1,37 @@
+package version
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/keboola/keboola-as-code/internal/pkg/filesystem"
+	"github.com/keboola/keboola-as-code/internal/pkg/log"
+	"github.com/keboola/keboola-as-code/internal/pkg/testfs"
+)
+
+func TestCheckManifestVersion_ValidVersion(t *testing.T) {
+	t.Parallel()
+	fs := testfs.NewMemoryFs()
+	assert.NoError(t, fs.WriteFile(filesystem.NewFile(`foo.json`, `{"version": 2}`)))
```

Comment:
Doplnene testy, ... hlavny problem bol v tom, ze ked field neexistoval, tak to predtym vracalo `invalid version "0"`, teraz je to uz fixnute.